src/components/modals/Send/steps/01-step-amount.js

```diff
@@ -128,9 +130,9 @@ export class StepAmountFooter extends PureComponent<
 
     try {
       const totalSpent = await bridge.getTotalSpent(account, transaction)
-      if (this._isUnmounted) return
+      if (syncId !== this.syncId) return
       const canBeSpent = await bridge.canBeSpent(account, transaction)
-      if (this._isUnmounted) return
+      if (syncId !== this.syncId) return
```

Comment:
future: dyou think we could make these syncId checks more generic, maybe using context API, or redux? would be awesome to avoid those checks in components